database-commons/src/main/java/io/cdap/plugin/db/batch/sink/ETLDBOutputFormat.java

```diff
@@ -89,8 +89,8 @@ public void close(TaskAttemptContext context) throws IOException {
           try {
             if (!emptyData) {
```

Comment:
I didn't understand this part. There was a 1.10 cherry pick. Why are we picking it up again for 1.7. Is there compatibility issue with different CDF versions

Reply:
Yes, we are releasing it with two different CDF versions.